packages/theme-base/src/css/component/text.scss

```diff
@@ -1,39 +1,47 @@
 .amplify-text {
   color: var(--amplify-components-text-color);
   display: block;
-  
+
+  @at-root b#{&},
```

Comment:
This is to fix the issue where `<Text as="span">` is being rendered as a block element. This results in a rule like:
```
b.amplify-text, em.amplify-text, etc... { display: inline; }
```